python/tests/integration/test_media_server_integration.py

```
@@ -295,16 +288,10 @@ def test_media_directory_request_does_not_list_or_500(make_server_project):
-        f"/media/photos must 404, got {response.status_code}"
-    )
-    assert trailing.status_code == 404, (
-        f"/media/photos/ must 404 (not list), got {trailing.status_code}"
-    )
+    assert response.status_code == 404, f"/media/photos must 404, got {response.status_code}"
+    assert trailing.status_code == 404, f"/media/photos/ must 404 (not list), got {trailing.status_code}"
     for body in (response.content, trailing.content):
-        assert b"a.txt" not in body and b"b.txt" not in body, (
-            "directory listing must not appear in the response body"
-        )
+        assert b"a.txt" not in body and b"b.txt" not in body, "directory listing must not appear in the response body"
```

Ruff (0.15.14)

[warning] Assertion should be broken down into multiple parts

(PT018)